Library/Homebrew/cmd/desc.rb

```diff
@@ -0,0 +1,59 @@
+require 'descriptions'
+require 'cmd/search'
+
+USAGE =<<EOM
+Print formula descriptions.
+
+Usage:
+
+    brew desc --decache             Delete the cache, if present
+    brew desc --recache             [Re]generate the cache
+    brew desc name [name ...]       Describe the named formula(e)
+    brew desc (-s|-n|-d) <string>   Search specifed fields for string
```

Comment:
Yeesh. Embarrassing. I’d fix that, except that I’m just going to take @xu-cheng’s suggestion and ditch the usage message entirely.